_data/master/navbars/getting-started.yml

```diff
@@ -135,6 +165,16 @@ toc:
   section:
   - title: Overview
     path: /getting-started/openstack/
+  - title: Requirements
+    section:
+    - title: System 
+      path: /getting-started/openstack/reqs/system
+    - title: Public cloud
+      section:
+      - title: Amazon Web Services
```

Comment:
If I go to http://0.0.0.0:4000/master/getting-started/, for some reason lots of the new sidebar structure is missing.  If I then go to http://0.0.0.0:4000/master/getting-started/openstack/installation/, the correct sidebar structure appears...  (weird!)